database/README.md

```diff
@@ -353,6 +357,14 @@ WHERE
 LIMIT 10000;
 ```
 
+### Creating a new geospatial layer
+
+When creating a new database table to hold feature data, polygon geometries should always be stored in a `Multipolygon` column type to avoid future issues. Any columns that will be populated with AGOL feature attribute data should exactly match the column names used in AGOL, except they should be lowercase.
+
+As a best practice, tables should always be configured with `created_at` and `updated_at` columns of type `timestamptz` with default `now()`. As well, the layer should be configured with an `on update` trigger to set the `updated_at` timestamp using the existing `set_updated_at_timestamp()` function.
```

Comment:
:pray: great documentation

Reply:
+1 - this (plus the existing docs) really help me to have a better mental model of the existing and future layers 🙏